runtime/drivers/salesforce/bulk.go

```diff
@@ -0,0 +1,292 @@
+package salesforce
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"encoding/xml"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	force "github.com/ForceCLI/force/lib"
+	"github.com/ForceCLI/force/lib/record_reader"
+	"go.uber.org/zap"
+)
+
+// Forceable represents an object which can talk to salesforce
+type Forceable interface {
```

Comment:
Either the library should expose an interface or we should directly use `force.Force`. I don't think its required here.

Reply:
Removed Forceable interface in de5424725d167b58dbbedb0273f12933eeeeccc0.